cea/analysis/costs/equations.py

```diff
@@ -17,6 +17,18 @@
 
 # Calculates the EQUIVALENT ANNUAL COSTS (1. Step PRESENT VALUE OF COSTS (PVC), 2. Step EQUIVALENT ANNUAL COSTS)
 def calc_opex_annualized(OpC_USDyr, Inv_IR_perc, Inv_LT_yr):
+    """
+    Calculate Annuitized Operating Expenditure (OPEX) -
+    Calculations based on http://fahmi.ba.free.fr/docs/Courses/2012%20HEC/FBA_FE_Chap1_time_value_derivation.pdf
+
+    :param OpC_USDyr: Operating Expenditure in USD per year
+    :param Inv_IR_perc: Interest Rate in percentage
+    :param Inv_LT_yr: Lifetime in years
+    :return: Annuitized Operating Expenditure in USD per year
+    """
+    if Inv_LT_yr == 0:
```

Comment:
Is this supposed to check for the interest rates instead of the lifetime variable?

Reply:
You're obviously right. Very bad oversight on my part. I just fixed it.